app/packs/src/apps/mydb/elements/details/reactions/schemeTab/Material.js

```diff
@@ -291,7 +291,7 @@ class Material extends Component {
       <td colSpan={colSpan} style={style}>
         <NumeralInputWithUnitsCompo
           precision={4}
-          bsStyle="success"
+          variant="success"
```

Comment:
bsStyle wasn't used in NumeralInputWithUnitsCompo before, so changing it to variant here would change the behaviour imo.

Reply:
Ok after looking at the file history, it looks like Joachim missed some occurrences when replacing bsStyle, so it should be allright